pages/linux/procs.md

```diff
@@ -7,18 +7,18 @@
 
 `procs`
 
-- Show information about processes, if the commands which started them contain `zsh`:
+- List information about processes, if the commands which started them contain `zsh`:
 
 `procs {{zsh}}`
 
-- Show information about all processes sorted by CPU time in [a]scending or [d]escending order:
+- List information about all processes sorted by CPU time in ascending or descending order:
 
 `procs {{--sortd|--sorta}} cpu`
 
-- Show information about processes with either a PID, command, or user containing (`zsh` or `firefox`):
+- List information about processes with either a PID, command, or user containing `firefox`:
```

Comment:
I'm not sure if I fully understand the semantics of this, but isn't it more like this

```suggestion
- List information about processes with either a PID, command, or user containing `41` or `firefox`:
```